libs/constrained-generators/src/Constrained/Base.hs

```diff
@@ -631,6 +631,13 @@ data Specification fn a where
   -- | Anything
   TrueSpec :: Specification fn a
 
+-- | return a MemberSpec or ans ErrorSpec depending on if 'xs' the null list or not
+memberSpecList :: [a] -> NE.NonEmpty String -> Specification fn a
```

Comment:
I would prefer the order of the arguments to be the other way around here, but that's a super minor thing.

Reply:
I tried that way first. But bringing the list as the first argument, mirrors the old code better.